resources/wrapper/godelw

```diff
@@ -161,23 +164,35 @@ GODEL_BASE_DIR=${GODEL_HOME:-$HOME/.godel}
 # determine OS
 OS=""
 EXPECTED_CHECKSUM=""
-case "$(uname)" in
-    Darwin*)
+case "$(uname)-$(arch)" in
```

Comment:
Let's use `$(uname -m)` here -- `arch` outputs `i386` on Intel Macs, including 64-bit ones (see https://apple.stackexchange.com/questions/140651/why-does-arch-output-i386 and verified by testing locally as well), while `uname -m` outputs `x86_64` on 64-bit Intel macs and `arm64` on M1 Macs.